traffic_ops/traffic_ops_golang/login/login.go

```diff
@@ -371,6 +378,13 @@ func OauthLoginHandler(db *sqlx.DB, cfg config.Config) http.HandlerFunc {
 			log.Errorf("checking local user: %s\n", err.Error())
 		}
 
+		_, dbErr := db.Exec(UpdateLoginTimeQuery, form.Username)
```

Comment:
The "Last Authenticated" time of a user should only be updated if the user successfully authenticates, which isn't checked until line 388 currently.